data/country.json

```diff
@@ -39,7 +39,38 @@
     "BM": "Bermuda",
     "BN": "Brunei Darussalam",
     "BO": "Bolivia",
-    "BR": "Brazil",
+    "BR": {
+        "name":"Brasil",
```

Comment:
Should this be Brazil?

Reply:
I think Brazil is the English spelling, while Brasil is the Brazilian Portuguese spelling.

> Brazil (/brəˈzɪl/ (About this sound listen); Portuguese: Brasil [bɾaˈziw]

[Wikipedia](https://en.wikipedia.org/wiki/Brazil)

Reply:
It should be Brazil to be translated in another moment as Brasil